exercises/03-sagas/instructions.md

```diff
@@ -32,6 +32,8 @@ For more info, please see [the instructions for running the exercise solutions](
 
 At the moment we gather all successfully paid orders and the only option we have is to process them in batches. Business is growing rapidly and we would like to process orders as soon as possible, not just at predefined intervals.
 
+Note: This is a hypothetical case. From experience we've seen a lot of developers implement batch processes to verify a certain state in the database. Sagas are 
```

Comment:
I haven't looked through the rest of the PR yet, but there's something missing here.